pyuvdata/uvdata/uvdata.py

```diff
@@ -6070,6 +6069,90 @@ def set_uvws_from_antenna_positions(self, update_vis=True):
         self.uvw_array = new_uvw
         return
 
+    def update_antenna_positions(
+        self, new_positions=None, delta_antpos=False, update_vis=True
+    ):
+        """
+        Update antenna positions and associated (meta)data.
+
+        This method will update the antenna positions for a UVData object, and
+        correspondingly will adjust the uvw-coordinates for each of the baselines, as
+        well as the phases of the visibilities (if requested).
+
+        Parameters
+        ----------
+        new_positions : dict
+            Dictionary containing new antenna positions, where the key is the antenna
+            number (which should match to an entry in UVData.antenna_numbers), and the
+            value is a 3-element array corresponding to the ECEF/MCMF position relative
+            to the array center.
+        delta_antpos : bool
+            When set to True, uvws are updated be calculating the difference between
+            the old and new antenna positions. Thiis option should be used with care,
```

Comment:
Thiis -> This

Reply:
Fixed in ac42d92d3a50e55de06e2aadf34d30236cccf9df. Reinstalling spell checker now...